usage/src/main/java/com/cloud/usage/UsageSanityChecker.java

```diff
@@ -170,14 +188,21 @@ protected void checkSnapshotUsage() {
     }
 
     protected void readLastCheckId(){
+        if (s_logger.isDebugEnabled()) {
+            s_logger.debug("reading last checked id for sanity check");
+        }
         try(BufferedReader reader = new BufferedReader(new FileReader(lastCheckFile));) {
             String lastIdText = null;
             lastId = -1;
-            if ((reader != null) && (lastIdText = reader.readLine()) != null) {
+            if ((lastIdText = reader.readLine()) != null) {
                 lastId = Integer.parseInt(lastIdText);
             }
         } catch (Exception e) {
-            s_logger.error("readLastCheckId:Exception:"+e.getMessage(),e);
+            String msg = String.format("error reading the LastCheckId reason: %s", e.getMessage());
+            s_logger.error(msg);
+            s_logger.debug(msg, e);
+        } finally {
+            s_logger.info(String.format("using %d as last checked id to start from in sanity check", lastId));
```

Comment:
```suggestion
            s_logger.info(String.format("Using %d as last checked id to start from in sanity check", lastId));
```

other logs messages if applicable

Reply:
This is excessive capitalisation to me, I won´t apply.